hommod_rest/services/domainalign.py

```diff
@@ -328,6 +328,22 @@ def getAlignment(self, _range, template):
 
         return self.pool[_range][templateID]
 
+    def getAlignmentYasara(self, _range, yasaraChain):
```

Comment:
Use `range_` instead of `_range`. Prefixing with an underscore is the pythonic way to denote a private member. It's better to postfix to avoid the clash with the `range` keyword.